### DISCUSSION TITLE
UWP Emgu4.5.1 install directions

### DISCUSSION THREAD

**Discussion by user20112** [emgucv/emgucv#417] [General]
Good evening,
im in the process of updating my emgu UWP installation from 3 to 4 and im running into some issues. there doesnt seem to be any UWP instructions at the link below.
https://emgu.com/wiki/index.php/Download_And_Installation#Windows
Currently i am recieving the runtime error below 
{"Unable to load DLL '__Internal' or one of its dependencies: The specified module could not be found. (Exception from HRESULT: 0x8007007E)"}
i am copying x64 86 arm and arm64  cvexterns into their respective folders on build and have confirmed they are there. other than that i have emgu.cv.platform.netstandard referenced and emgu.cv.bitmap referenced. is there any other reference im missing or suggestion on how to debug this ?

**Comment by emgucv** [comment]
Instruction has been updated. 

If you are using nuget, once you setup the commercial nuget repository, all you need to do is to add the nuget package "Emgu.CV.runtime.uwp" into your project. Please remove all other Emgu CV refenferences before you installed the uwp runtime package.

Note the Emgu.CV.Bitmap can be installed but will not work under UWP. Bitmap required GDI+, but GDI+ is not available under UWP.

**Reply by user20112** [reply]
Thanks so much ! I figured it was something small i was missing xD sorry for the several questions recently i usually go to the old forum answers but the old forum seems to have been shutdown.
